tests/plugin_http_api_test.py

```diff
@@ -113,6 +113,7 @@ def test_ChainApi(self) :
         self.activateAllBuiltinProtocolFeatures()
         allProtocolFeatures = self.nodeos.getSupportedProtocolFeatures()
         allFeatureDigests = [d['feature_digest'] for d in allProtocolFeatures]
+        allFeatureCodenames = [s['specification'][0]['value'] for s in allProtocolFeatures]
```

Comment:
It would be nice if we could make sure we're looking at `name == builtin_feature_codename` before assuming it's the `value` we want.. dunno if complexity is worth it tho. Fine as is imo

Reply:
Thanks for the comments. I will add the check.